src/Microsoft.AspNet.Security.Twitter/TwitterAuthenticationHandler.cs

```diff
@@ -115,12 +115,12 @@ protected override AuthenticationTicket AuthenticateCore()
 
                 await Options.Notifications.Authenticated(context);
 
-                return new AuthenticationTicket(context.Identity, context.Properties);
+                return new AuthenticationTicket(context.Identity, context.Properties, Options.AuthenticationScheme);
             }
             catch (Exception ex)
             {
                 _logger.WriteError("Authentication failed", ex);
-                return new AuthenticationTicket(null, properties);
+                return new AuthenticationTicket((ClaimsPrincipal)null, properties, Options.AuthenticationScheme);
```

Comment:
Can we add an overload of AuthenticationTicket with just `properties` and `scheme` that initializes principal internally to null as seem to use this often?